src/atoms/Button/Button.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+
+const Pulse = keyframes`
+    0% {
+        transform: scale(0.95);
+        box-shadow: 0 0 0 0 rgba(0, 0, 0, 0.5);
+    }
+
+    50% {
+        transform: scale(1);
+        box-shadow: 0 0 0 10px rgba(0, 0, 0, 0);
+    }
+
+    100% {
+        transform: scale(0.95);
+        box-shadow: 0 0 0 0 rgba(0, 0, 0, 0);
+    }
+`;
+
+const StyledButton = styled.button`
```

Comment:
Avoid mixing your styled components with the main component. I suggest to extract the styled components into its own `styled.js` file. This way you will have a more cleaner component, as the styled components tend to get bigger due to CSS.